python_docs_theme/layout.html

```diff
@@ -137,12 +137,9 @@ <h3>{{ _('Navigation') }}</h3>
 {% block footer %}
     <div class="footer">
     &copy; {% if theme_copyright_url or hasdoc('copyright') %}
-      <a href="{% if theme_copyright_url %}{{ theme_copyright_url }}{% else %}{{ pathto('copyright') }}{% endif %}">
-    {% endif %}
-    {% trans %}Copyright{% endtrans %}
-    {% if theme_copyright_url or hasdoc('copyright') %}
-      </a>
-    {% endif %} {{ copyright|e }}.
+      {# Wrapping the below link introduces spurious whitespace #}
```

Comment:
We can keep the wrapped version and add minus symbols to tell it to delete whitespace from that side:

```
    {%- endif %}
    {%- trans %}BCopyright{% endtrans %}{% if theme_copyright_url or hasdoc('copyright') %}</a>
```